src/plugins/gauge/components/Gauge.vue

```diff
@@ -343,14 +390,28 @@ export default {
         dialLowLimitDeg() {
             return this.percentToDegrees(this.valToPercent(this.limitLow));
         },
+        meterOorRangeIndicatorAspectRatio() {
+            return this.typeMeterVertical ? 'xMidYMax meet' : 'xMinYMid meet';
+        },
+        meterTextBaseline() {
+            return this.typeMeterVertical ? 'auto' : 'middle';
+        },
         curValViewBox() {
-            const DIGITS_RATIO = 10;
-            const VIEWBOX_STR = '0 0 X 15';
+            const DIGITS_RATIO = 3;
+            const VIEWBOX_STR = '0 0 X 10';
 
             return VIEWBOX_STR.replace('X', this.digits * DIGITS_RATIO);
         },
+        rangeFontSize() {
+            const CHAR_THRESHOLD = 3;
+            const START_PERC = 8.5;
+            const REDUCE_PERC = 0.8;
+            const RANGE_CHARS_MAX = Math.max(this.rangeLow.toString().length, this.rangeHigh.toString().length);
```

Comment:
can `rangeLow` ever be null?

Reply:
Never in a normal case; this value is set by default and passed in from GaugePlugin.js, and should be handled there.

Reply:
👍